### ISSUE TITLE
French screenshots are pulled for ScummVM games regardless of region preference

### ISSUE THREAD

**Issue by cameronhimself** [muldjord/skyscraper#233]
**Describe the bug**
Regardless of any region options set, the French screenshots are pulled from ScreenScraper for many (all?) ScummVM games.

**To Reproduce**
This has occurred for (I think) every ScummVM game I've attempted to scrape, but to keep it simple I'm going to focus on Day of the Tentacle (DotT). These steps can be performed without a copy of the game due to the way ScummVM games are detected.
1. Observe on the [ScreenScraper page for DotT](https://www.screenscraper.fr/gameinfos.php?gameid=99777&action=onglet&zone=gameinfosmedias) that there is a screenshot with English text for the "monde" region. My French isn't what it could be, but I'm pretty certain that "monde" means "world".
1. For consistency of results, ensure `regionPrios` aren't set in `config.ini`. In my experience this setting hasn't made any difference, but I'd like to be safe.
1. Again, for consistency, clear caches for scummvm: `Skyscraper -p scummvm --cache purge:all`
1. Create the following file structure in your current working directory: `tentacle/tentacle.scummvm`, with `tentacle.scummvm` being a text file containing only the exact string "tentacle" (minus quotes): `mkdir tentacle && echo tentacle > tentacle/tentacle.scummvm`
1. `Skyscraper -i $(pwd)/tentacle --region wor -p scummvm -s screenscraper`
1. `Skyscraper -i $(pwd)/tentacle --region wor -p scummvm`
1. Open `tentacle/media/screenshots/tentacle.png` in your favorite image viewer and note the French interface text. Specifically, [this image](https://i.imgur.com/XGFpKMw.png) (unless someone has uploaded a different image to ScreenScraper since I opened this issue).

These steps can be repeated for other regions with the same result. I've tried `us`, `uk`, `eu`, `ss`, and even `sp` to see if I could pull the available Spanish screenshot. They all result in the French screenshot linked above or no screenshot at all.

**Special circumstances**
I'm generating configuration for the Pegasus frontend on Android. I'm running Skyscraper on my Windows PC (through Ubuntu on the Windows Subsystem for Linux) to generate configuration to be copied to the Android device.

**Terminal output**
```
○ → Skyscraper -i $(pwd)/tentacle --region wor --verbosity 3 -p scummvm -s screenscraper
------------------------------------------
Running Skyscraper v3.5.5 by Lars Muldjord
------------------------------------------
Platform:           'scummvm'
Scraping module:    'screenscraper'
Extra:              'am start --user 0
  -n com.retroarch/.browser.retroactivity.RetroActivityFuture
  -e ROM {file.path}
  -e LIBRETRO /data/data/com.retroarch/cores/scummvm_libretro_android.so
  -e CONFIGFILE /storage/emulated/0/Android/data/com.retroarch/files/retroarch.cfg
  -e IME com.android.inputmethod.latin/.LatinIME
  -e DATADIR /data/data/com.retroarch
  -e APK /data/app/com.retroarch-1/base.apk
  -e SDCARD /storage/emulated/0
  -e EXTERNAL /storage/emulated/0/Android/data/com.retroarch/files
  --activity-clear-top'
Input folder:       '/mnt/d/media/roms/skyscraper/tentacle'
Game list folder:   '/mnt/d/media/roms/skyscraper/tentacle'
Covers folder:      '/mnt/d/media/roms/skyscraper/tentacle/media/covers'
Screenshots folder: '/mnt/d/media/roms/skyscraper/tentacle/media/screenshots'
Wheels folder:      '/mnt/d/media/roms/skyscraper/tentacle/media/wheels'
Marquees folder:    '/mnt/d/media/roms/skyscraper/tentacle/media/marquees'
Cache folder:       'cache/scummvm'

Forcing 1 threads as this is the anonymous limit in the ScreenScraper scraping module. Sign up for an account at https://www.screenscraper.fr and support them to gain more threads. Then use the credentials with Skyscraper using the '-u user:password' command line option or by setting 'userCreds="user:password"' in '/home/USER/.skyscraper/config.ini'.

Reading and parsing quick id xml, please wait... Done!
Reading and parsing resource cache, please wait... Done!
Successfully parsed 0 resources!

Resource cache stats for selected platform:

Looking for optional 'priorities.xml' file in cache folder... Found!
Priorities loaded successfully!



Starting scraping run on 1 files using 1 threads.
Sit back, relax and let me do the work! :)

cover: '456952' > '200784', choosing resize for optimal result!
marquee: '477844' > '70531', choosing resize for optimal result!
#1/1 (T1) Pass 1 ---- Game 'tentacle' found! :) ----
Scraper:        screenscraper
From cache:     NO
Search match:   100 %
Compare title:  'tentacle'
Result title:   'Day of the Tentacle' ()
Platform:       'ScummVM' ()
Release Date:   '1993-01-01' ()
Developer:      'LucasArts' ()
Publisher:      'LucasArts' ()
Players:        '1' ()
Ages:           '1+' ()
Tags:           'Adventure / Point and Click, Adventure' ()
Rating (0-1):   '0.7' ()
Cover:          YES ()
Screenshot:     YES ()
Wheel:          YES ()
Marquee:        YES ()

Description: ()
'Dr. Fred's mutated purple tentacle is out to take over the world. And only you can stop him. Travel through time with three outrageous characters in this wacky graphic adventure which features cartoon-style animation and over 100 zany sound effects.'

'screenscraper' requests remaining: 9935

Debug output:
Tried with: 'crc=AC24126F&md5=1C5C41D6AB29AFC80D3BDDA10FC489FE&sha1=B82D1F8F6CBCA6C5D1582FA022FF8E288B3B352F&romnom=tentacle.scummvm&romtaille=9'
Platform: scummvm

#1/1, (1/0)
Elapsed time   : 00:00:06
Est. time left : 00:00:00

---- Resource gathering run completed! YAY! ----
Writing quick id xml, please wait... Done!
Writing 14 (14 new) resources to cache, please wait... Done!

---- And here are some neat stats :) ----
Total completion time: 00:00:06

Average search match: 100%
Average entry completeness: 100%

Total number of games: 1
Successfully processed games: 1
Skipped games: 0 (Filenames saved to '/home/USER/.skyscraper/skipped-scummvm-screenscraper.txt')
```

**Technical information**
 - Skyscraper version: 3.5.5
 - Internet connection: adsl
 - OS: Windows 10
 - Version 1803, OS build 17134.1425.

**Comment by muldjord** [comment]
First off, thank you for being so thorough in your description. It makes my job so much easier. The solution to this is easier than I thought. For some reason I never added region support for screenshots. I guess I must've concluded that screenshots aren't region specific... But of course they can be! So I'll add this in the next release, which will fix this issue. Thank you for reporting it and taking the time to properly describe it.

EDIT: This is now fixed and will be in 3.5.6

**Comment by cameronhimself** [comment]
Awesome! Thank you for being such a responsive and involved maintainer @muldjord 👍
